src/test/java/teammates/test/cases/webapi/UpdateFeedbackResponseActionTest.java

```diff
@@ -25,6 +27,16 @@
  */
 public class UpdateFeedbackResponseActionTest extends BaseActionTest<UpdateFeedbackResponseAction> {
 
+    private StudentAttributes student1InCourse1;
+    private StudentAttributes student2InCourse1;
+    private StudentAttributes student5InCourse1;
```

Comment:
This is only utilized in `testAccessControl_studentAccessOtherTeam_inaccessible`

Reply:
> It's also needed in `testAccessControl_giverNotModeratedStudent_inaccessible` because it's required by `testModerateResponse` which in turn requires two additional students of which neither should be the moderated person

Same as utilisation